packages/javascript/bh-shared-ui/src/hooks/useAssetGroupTags/useAssetGroupTags.tsx

```diff
@@ -292,8 +292,8 @@ export const useSelectorMembersInfiniteQuery = (
     }>({
         queryKey: privilegeZonesKeys.membersByTagAndSelector(tagId!, selectorId, sortOrder, environments),
```

Comment:
Is `selectorId` staying the same or does it need to be `ruleId`?

Reply:
I can likely go either way on this but I think keeping the internal logic selector and the front facing as "rule" is likely the way to go for consistency and maintainability, especially given the api is still selector